netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +159,31 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages has to throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test(expected = HttpException.class)
+    public void exceptionIfBoutNotFound() throws IOException {
+        Messages messages = null;
```

Comment:
@2686747 we strictly try to avoid using `null`, see [here](http://www.yegor256.com/2014/05/13/why-null-is-bad.html).